src/datepicker/datepicker.component.ts

```diff
@@ -280,6 +279,12 @@ export class DatePicker implements OnDestroy, OnChanges, AfterViewChecked {
 		const daysContainer = document.querySelectorAll(".flatpickr-days");
 		const dayContainer = document.querySelectorAll(".flatpickr-day");
 
+		Array.from(calendarContainer)
+			.forEach(calendar => {
+				calendar.removeEventListener("click", event => event.stopPropagation());
+				calendar.addEventListener("click", event => event.stopPropagation());
```

Comment:
In order to remove it you'll have to add the listener as a property - something like `protected preventCalendarClose = event => event.stopPropagation();` and then pass that to add/remove listener